insights/specs/datasources/kernel_module_list.py

```diff
@@ -5,16 +5,21 @@
 from insights.core.exceptions import SkipComponent
 from insights.core.filters import get_filters
 from insights.core.plugins import datasource
+from insights.parsers.lsmod import LsMod
 from insights.specs import Specs
 
 
-@datasource(HostContext)
+@datasource(LsMod, HostContext)
```

Comment:
If a parser is required by a datasource, the data source should be `pre-loaded` while the collection, please add it to the "plugins.configs" section of "insights.collect.default_manifest" module, and set its `enable` as true.

Reply:
@xiangce - Updated, please check